x-pack/plugin/eql/src/test/resources/queryfolder_tests.txt

```diff
@@ -239,6 +239,13 @@ process where cidrMatch(source_address, "10.0.0.0/8", "192.168.0.0/16")
 {"terms":{"source_address":["10.0.0.0/8","192.168.0.0/16"]
 ;
 
+cidrMatchFunctionTwoWithOr
+process where cidrMatch(source_address, "10.0.0.0/8") or cidrMatch(source_address, "192.168.0.0/16")
+;
+{"bool":{"must":[{"term":{"event.category":{"value":"process"
+{"terms":{"source_address":["10.0.0.0/8"],"boost":1.0}},{"terms":{"source_address":["192.168.0.0/16"]
```

Comment:
Indeed, it is relevant to have both `terms` queries in there and the `must` for `event.category`, **but there has to be a `bool` `should` as well.**
The query should translate as a `bool` with two `must` statements:
- one is `term` on `event.category`
- the other is a `bool` with two `should` statements where `terms` on the `source_address` is used.

Reply:
Updated